src/main.ts

```diff
@@ -151,6 +151,10 @@ export default class PointApiImpl implements PointApi {
     headers?: object,
   ) {
     const jwt = await this.authManager.getJwt();
+    
+    if (this.authManager.isUserActive() === false) {
+      throw new Error("User inactive");
```

Comment:
I'm throwing an error if the user is inactive. Not ideal but works for now.

Reply:
@przxmek what do you think the ideal flow would be?

Reply:
Probably the client (account-management & js-sdk) should now that the user is inactive prior to making "blind requests" and finding out as they get an error instead of response.

Now it looks like:
1. /auth --> ok (no info to client that user is inactive)
2. /account --> error (User Inactive) _That's when I redirect them to "waitlist info" aka getting-started page_
3. /custom/suggestions --> error (User Inactive)
4. _anything else if_ --> error (User Inactive)

Proposal:
1. /auth --> ok_but_inactive (client now that user is inactive and will adjust behavior immediately)
2. Don't send any more requests that will fail for inactive user; redirect to "waitlist info" aka getting-started page_

It feels that it would require even more changes and doesn't seem like worth doing now tbh.